group/edwards25519/ge.go

```diff
@@ -547,16 +547,16 @@ func Ge25519HasSmallOrder(s []byte) int32 {
 //
 // Checks whether group element s is less than p
 func Ge25519IsCanonical(s []byte) int32 {
-	var c uint8
-	var d uint8
+	var c = 0
```

Comment:
```suggestion
	c := 0
```

same for `var d = 0`

Reply:
OK, I'll change these.